pkg/virt-operator/resource/generate/components/validatingadmissionpolicy.go

```diff
@@ -147,8 +148,21 @@ func NewHandlerV1ValidatingAdmissionPolicy(virtHandlerServiceAccount string) *ad
 					Name:       "newAnnotations",
 					Expression: "object.metadata.annotations",
 				},
+				{
+					Name:       "requestMatchNode",
+					Expression: "object.metadata.name == request.userInfo.extra['authentication.kubernetes.io/node-name'][0]",
```

Comment:
is this `[0]` future proof?

Reply:
Ok, i don't insist, but it's just strange. How do we even know that this is going to remain this way?